pkg/utils/utils.go

```diff
@@ -212,7 +212,6 @@ func GetObjectMetaData(obj interface{}) metaV1.ObjectMeta {
 		Annotations:                unstructuredObject.GetAnnotations(),
 		OwnerReferences:            unstructuredObject.GetOwnerReferences(),
 		Finalizers:                 unstructuredObject.GetFinalizers(),
-		ClusterName:                unstructuredObject.GetClusterName(),
```

Comment:
https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/object-meta/#Ignored

> Deprecated: ClusterName is a legacy field that was always cleared by the system and never used; it will be removed completely in 1.25.